assets/src/edit-story/karma/fixture/fixture.js

```diff
@@ -416,12 +417,17 @@ class APIProviderFixture {
         const filterBySearchTerm = searchTerm
           ? ({ alt_text }) => alt_text.includes(searchTerm)
           : () => true;
+        // Generate 7*6=42 items, 3 pages
+        const clonedMedia = Array(6)
+          .fill(getMediaResponse)
+          .flat()
+          .map((media, i) => ({ ...media, id: i + 1 }));
         return asyncResponse({
-          data: getMediaResponse
-            .slice((pagingNum - 1) * 20, 20)
+          data: clonedMedia
+            .slice((pagingNum - 1) * MEDIA_PER_PAGE, pagingNum * MEDIA_PER_PAGE)
```

Comment:
It took me a good 15 minutes to spot it 😅  @barklund

Reply:
You never know when slice vs splice arguments (index based vs item count) will get you 🐛 .